pycm/pycm_curve.py

```diff
@@ -0,0 +1,152 @@
+# -*- coding: utf-8 -*-
+"""Curve module."""
+from __future__ import division
+from .pycm_error import pycmCurveError
+from .pycm_util import threshold_func, thresholds_calc, isfloat
+from .pycm_param import *
+from .pycm_obj import ConfusionMatrix
+import numpy
+
+
+class Curve:
+    """
+    Curve class.
+
+    >>> cv = Curve(actual_vector = np.array([1, 1, 2, 2]), probs = np.array([[0.1, 0.9], [0.4, 0.6], [0.35, 0.65], [0.8, 0.2]]), classes=[2,1])
+    >>> cm.classes
+    [2,1]
```

Comment:
`[2, 1]`

Reply:
fixed in https://github.com/sepandhaghighi/pycm/pull/457/commits/b49a1624972ceb4b627263ee299fb25350e8319d